### DISCUSSION TITLE
ECHIPA EXPLORATORII

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#104] [TEAMs]
Aici sunt membrii echipei EXPLORATORII

**Comment by sebiboga** [comment]
Larisa Palimaru
role: scraper developer
discord: LarisaM16

**Reply by LarisaM16** [reply]
1 Manual tester

**Reply by LarisaM16** [reply]
1 automation tester

**Reply by LarisaM16** [reply]
1 team manager

**Comment by LarisaM16** [comment]
Simona Nemtanu
role: Manual Tester
discord: simonanemtanu_21933




Rares Irimus
role: Automation Tester
discord: RaresCode
